### PR TITLE
apache beam now supports python 3

### PR DESCRIPTION
**Pull request by kousun12**
https://beam.apache.org/roadmap/python-sdk/#python-3-support

python 3.5 supported since 2.11.0 and python 3.6/3.7 supported since 2.14.0

this pr removes the python 2 conditionals for beam

### CODE REVIEW

File: magenta/models/score2perf/datagen_beam.py [hunk 1]
```diff
@@ -42,10 +41,11 @@
 # TODO(iansimon): this should probably be defined in the problem
 SCORE_BPM = 120.0
 
-FLAGS = flags.FLAGS
 flags.DEFINE_string(
     'pipeline_options', '',
     'Command line flags to use in constructing the Beam pipeline options.')
+FLAGS = flags.FLAGS
+FLAGS(sys.argv)
```
Comment (iansimon) [inline magenta/models/score2perf/datagen_beam.py]: Why is this needed?

Comment (kousun12) [reply magenta/models/score2perf/datagen_beam.py]: without it, usage later complains that we're trying to use the flag before it's been declared for some reason, and the specs fail. but actually it looks like the `tf.app.flags.FLAGS` wrapper rather than importing from `absl`. just made that change
